### ISSUE TITLE
Implement "after-controller" middleware

### ISSUE THREAD

**Issue by RobyFerro** [RobyFerro/go-web-framework#1]
Middleware should operate even after the controller logic.
Checks Gorilla MUX documentation and find a way to implement into the framework.

Maybe something like the below snippet:

```
http.ListenAndServe(":8000", runsAfter(runsbefore(r)))

func runsafter(h http.Handler) http.Handler {
  fn := func(w http.ResponseWriter, r *http.Request) {
    h.ServeHTTP(w, r)
    w.Write([]byte("run after, "))
  }

  return http.HandlerFunc(fn)
}
```

**Comment by RobyFerro** [comment]
This issue should not be closed by 75a6475.

**Comment by RobyFerro** [comment]
The solution is simple as below:
```
// Logging log every actions printing used route
func (Middleware) Logging(next http.Handler) http.Handler {
	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
		next.ServeHTTP(w, r)
                log.Print("Run after")
	})
}
```
I close this issue
